hypothesis-python/README.rst

```diff
@@ -56,3 +56,7 @@ If you want to hear from people who are already using Hypothesis, some of them `
 about it <https://hypothesis.readthedocs.io/en/latest/endorsements.html>`_.
 
 If you want to create a downstream package of Hypothesis, please read `these guidelines for packagers <https://hypothesis.readthedocs.io/en/latest/packaging.html>`_.
+
+While Hypothesis has never been subject to a security issue, please use
+the [Tidelift security contact](https://tidelift.com/security) to report
```

Comment:
Again, not keen on including things like this until we formally decide what to do about tidelift.

Reply:
OK, I've taken it out of this PR so we don't need to delay merging.

I'm generally in favor of Tidelift as I like their business model - there's genuine business value that a donation couldn't buy, without anything onerous on the maintainer side.  Basically if you're comfortable with adding this kind of security policy link and a tidelift link via the GitHub sponsor button (possibly alongside individual sponsorships, which are finally out of beta) we can go ahead; otherwise I'll drop it until you tell me otherwise :slightly_smiling_face: